manifests/pcidevices-operator.yaml

```diff
@@ -28,6 +28,9 @@ rules:
   - apiGroups: ["apiregistration.k8s.io"]
     resources: ["apiservices"]
     verbs: [ "get", "watch", "list", "update", "create", "delete" ]
+  - apiGroups: ["kubevirt.io"]
+    resources: ["kubevirts"]
+    verbs: [ "get", "update" ]
```

Comment:
Thanks @ibrokethecloud for creating the chart PR: https://github.com/harvester/charts/pull/123